updates/17.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE einzelbeleg DROP FOREIGN KEY einzelbeleg_QuelleGattungID;
+-- this is necessary because the id column contains 0 values. Otherwise an error appears
+ALTER TABLE einzelbeleg ADD CONSTRAINT einzelbeleg_QuelleGattungID FOREIGN KEY (QuelleGattungID) REFERENCES selektion_quellengattung(ID) ON UPDATE CASCADE;
+SET SESSION sql_mode = CONCAT(@@SQL_MODE, ',NO_AUTO_VALUE_ON_ZERO');
+ALTER TABLE selektion_quellengattung MODIFY COLUMN ID INT NOT NULL AUTO_INCREMENT;
```

Comment:
The sql mode seems to be OK now, but if i execute all statements in the given order on my machine it fails:
`Cannot change column 'ID': used in a foreign key constraint 'einzelbeleg_QuelleGattungID' of table 'neg.einzelbeleg'`

Reply:
I had problems to reproduce the issue on my machine.
But i have changed the order of the statements.
On my machine it is working.
Can you please try it again